frame/referenda/src/lib.rs

```diff
@@ -83,15 +83,18 @@ use sp_std::{fmt::Debug, prelude::*};
 mod branch;
 mod types;
 pub mod weights;
-use branch::{BeginDecidingBranch, OneFewerDecidingBranch, ServiceBranch};
-pub use pallet::*;
-pub use types::{
-	BalanceOf, CallOf, Curve, DecidingStatus, DecidingStatusOf, Deposit, InsertSorted,
-	NegativeImbalanceOf, PalletsOriginOf, ReferendumIndex, ReferendumInfo, ReferendumInfoOf,
-	ReferendumStatus, ReferendumStatusOf, ScheduleAddressOf, TallyOf, TrackIdOf, TrackInfo,
-	TrackInfoOf, TracksInfo, VotesOf,
+
+use self::branch::{BeginDecidingBranch, OneFewerDecidingBranch, ServiceBranch};
+pub use self::{
```

Comment:
why self?

Reply:
Adding `self` or `crate` makes it easier to distinguish `module` name from `external crate` name, I think